fuse_models/include/fuse_models/parameters/odometry_2d_publisher_params.h

```diff
@@ -82,13 +82,8 @@ struct Odometry2DPublisherParams : public ParameterBase
     nh.param("scale_process_noise", scale_process_noise, scale_process_noise);
     nh.param("velocity_norm_min", velocity_norm_min, velocity_norm_min);
 
-    double tf_cache_time_double = tf_cache_time.toSec();
-    nh.getParam("tf_cache_time", tf_cache_time_double);
-    tf_cache_time.fromSec(tf_cache_time_double);
-
-    double tf_timeout_double = tf_timeout.toSec();
-    nh.getParam("tf_timeout", tf_timeout_double);
-    tf_timeout.fromSec(tf_timeout_double);
+    fuse_core::getPositiveParam(nh, "tf_cache_time", tf_cache_time, false);
```

Comment:
I've also used `fuse_core::getPositiveParam` for these two, that didn't required the input to be positive.

I believe they must be positive, and I've allowed `0.0` because:
* tf_cache_time: A TF buffer with a time cache that only spans for `0.0` seconds probably doesn't make much sense. I believe in this case only the latest transform is kept in the buffer.
* tf_timeout: A `lookupTransform` with `0.0` timeout will simply not wait for the transform to be available.

If you don't agree with any of this, I'm happy to change it.

Reply:
Looks good to me.